src/station22.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Station 22</title>
+    <script type="text/javascript">
+        function getData() {
+            const argumentArray = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n'];
+            return
+        }
+
+        // ↓こちらの関数は触らないこと
+        function hogehoge(s1, s2, s3, s4, s5, s6, s7, s8, s9, s10, s11, s12, s13, s14) {
+          return s1 + s2 + s3 + s4 + s5 + s6 + s7 + s8 + s9 + s10 + s11 + s12 + s13 + s14
+        }
```

Comment:
今回のPRとは関係ないところだけれど気になったので...．↓じゃない理由って何かあるんですかね？

```suggestion
        function hogehoge(...args) {
          return args.join('')
        }
```

Reply:
僕はなにも聞いてないのでわからないです。。

Reply:
JSは完全に人に任せたのでほかもとんでもないことになってそう...．

Reply:
oh...